There are some potential issues when running the validate.py

# dropped 2 coming soon locations
